todo-app/src/Status.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
 
 export default class Status extends React.Component{
-    render() {
+    constructor(props) {
+        super(props);
+        this.state = {isDone: false};
```

Comment:
If you add an `isDone` `boolean` property to each item in `todo-app/src/TasksDB.js`, then you can pass it as prop to `Status` component and set the state.

```js
this.state = {isDone: this.props.isDone};
```